R/survival.R

```diff
@@ -0,0 +1,125 @@
+
+#' Function for Survival Analysis view.
+#' It does survfit and survdiff.
+#' @export
+exp_survival <- function(df, time, status, start_time = NULL, end_time = NULL, time_unit = "day", cohort = NULL, cohort_func = NULL, ...){
+  validate_empty_data(df)
+
+  grouped_col <- grouped_by(df)
+
+  if (!is.null(substitute(cohort))) {
+    cohort_col <- col_name(substitute(cohort))
+    if (!is.null(cohort_func)) {
+      df[[cohort_col]] <- extract_from_date(df[[cohort_col]], type=cohort_func)
+    }
+  }
+  else {
+    cohort_col <- "1"
+  }
+
+  # substitute is needed because time can be
+  # NSE column name and it throws an evaluation error
```

Comment:
What is NSE?

Reply:
It stands for Non-Standard Evaluation. Added comment.